torch_tvm/custom_tvm_ops/topi/custom_topi_ops.cc

```diff
@@ -0,0 +1,248 @@
+#include <topi/generic/injective.h>
```

Comment:
why are we merging everything into one file? this will probably make rebasing hard going forward.  you could just copy paste the custom_layer_norm_init.cc file and replace it with your stuff

Reply:
Hmm, I actually prefer a centralized place to store all these registration. @yinghai I think c2 does the same thing, right?

Reply:
Yeah, just a place to see how many ops we have.